roles/container-engine/cri-o/vars/v1.29.yml

```diff
@@ -1,8 +1,8 @@
 ---
 # cri-o binary files
 crio_bin_files:
-  - conmon
-  - conmonrs
+  - crio-conmon
```

Comment:
As kubespray supports multiple kubernetes/crio version I think we would need to (unfortunately) have some logic to handle the previous versions differently right?